content/articles/JavaSript-Shopping-Cart/index.md

```diff
@@ -0,0 +1,268 @@
+### Introduction
+Building a shopping cart can be done in an assortment of ways. In this tutorial, I'll show you how to make a shopping cart using arrays and objects.
+
+### The approach
+1. You initially have a curated list of shopping items.
+2. Each item has a counter.
+3. You decrease or increase the counter on adding the item to cart.
+
+We will use the Javascript event listeners to handle these activities.
+
+### Preferences
+To follow along with this tutorial, you are required to have an understanding of the following JavaScript methods:
+
+1. [Array.push()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push)
+2. [Array.pop()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop)
+3. [EventTarget.addEventListener()](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener)
+
+### The base html
+Create a new file `index.html` in the working directory. In your code editor, open the `index.html` file and paste the code below.
+
+```html
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Document</title>
+
+<div id="wrapper">
+   <div id="container">
+       <h1>Shopping list</h1>
+       <div id="items"></div>
+   </div>
+    <div id="container">
+        <h1>Cart</h1>
+        <div id="cart"></div>
+    </div>
+</div>
+<script src="./script.js"></script>
+```
+
+### Creating the items library
+Create a new file `script.js` in the working directory.
+
+In your code editor, open the `script.js` file and add the code below.
+
+```javascript
+let items_array = [
+    { "name": "carrots", "id": 1, count: 1 },
+    { "name": "spinach", "id": 2, count: 1 },
+    { "name": "cookies", "id": 3, count: 1 },
+    { "name": "lettuce", "id": 4, count: 1 },
+    { "name": "avocado", "id": 5, count: 1 }
+];
+
+let cart = [];
+```
+
+The `items_array` is an array of objects which will be displayed for selection. A `name` and `id` property is given to each object.
+The `items_array` also has a `count` property, which is the number of items which will be used in the shopping cart.
+
+The cart is an empty array, which will be used to store the items which will be selected from the shopping cart.
+
+### Creating the base functions
+Let's make a few functions to make our lives a little bit easier. 
+
+In your code editor, open the `script.js` file and add the code below.
+
```

Comment:
@twish-254 This statement is redundant throughout your article. Please replace all the appearances after the first one with something similar to this: `Add the following code to script.js`